src/js/components/settings/settings.js

```diff
@@ -84,7 +84,7 @@ export const Settings = ({
         }
       });
     } else {
-      const notStripePromise = {};
+      const notStripePromise = new Promise(resolve => setTimeout(resolve, 700));
```

Comment:
from nothing to `700` -_-

Reply:
just anything that's not nothing - usually the stripe request should be faster (they are in a `Promise.race` 🏎️  after this) and at these times it still is considered to not be perceived by a user 👍 (although 700 is on the upper end of what we use in the codebase)